modules/administration-guide/pages/extensions-for-microsoft-visual-studio-code-open-source.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+:description: Extensions for Microsoft Visual Studio Code - Open Source
+:keywords: extensions, vs-code, vsx, open-vsx, marketplace
+:navtitle: Extensions for Microsoft Visual Studio Code - Open Source
+//:page-aliases:
+
+[id="extensions-for-microsoft-visual-studio-code-open-source"]
+= Extensions for Microsoft Visual Studio Code - Open Source
+
+To manage extensions, the link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source IDE] cloud editors use one of these link:https://open-vsx.org/about[Open VSX] registries:
```

Comment:
```suggestion
To manage extensions, the link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] IDEs that load in workspaces use one of these link:https://open-vsx.org/about[Open VSX] registries:
```

Reply:
why "a" ? there's only one code-rhel8 image per release per deployment.

Reply:
"to manage exetensions, the editor uses an openvsx registry" is a lot cleaner IMHO.

Reply:
"IDE that loads in workspaces" == "editor" :D 

why so wordy?

https://imgflip.com/i/77vsq9 :D

Reply:
```suggestion
To manage extensions, the editor uses one of these link:https://open-vsx.org/about[Open VSX] registries:
```

Reply:
> not typical for editors to load in workspaces

ok now I'm confused. 

if IDE = editor, and 
workspace = filesystem w/ an editor (including extensions) and 1 or more checked out projects, then

surely every workspace has an editor. 

If not, then what would you use to edit files in the workspace? the terminal?

Do we have a "no editor" option when loading a workspace?

Reply:
On second thought, maybe I should add the VS Code name back into my suggestion because otherwise it's reads to general.